content/management/configuration-api/v3.2/index.mdx

```diff
@@ -1932,6 +1932,10 @@ Here's what you need to know about **webhooks**:
 
 ### Register Webhook
 
+Number of webhooks that you can register is limited and depends on authorization method:
+- while using PAT (using diffrent PATs *doesn't* rise this limit): 10 for every action
```

Comment:
Ar you shure that we distinguish `pats`? AFAIR we just use `pat` in db entries

Reply:
by "using diffrent PATs *doesn't* rise this limit" I meant that we are not distinguishing PATs -> clientId is "personal_access_token" when someone is using PAT

Reply:
Ok, I read it wrong way xD